src/drivers/sifive,spi0.c

```diff
@@ -0,0 +1,232 @@
+/* Copyright 2018 SiFive, Inc */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#include <metal/drivers/sifive,spi0.h>
+#include <metal/io.h>
+
+/* Register byte offsets */
+#define SPI_REG_SCKDIV  0x000 
```

Comment:
Even though they are local, can we also prefix these with METAL_, like other drivers.